.github/workflows/create-release-pr.yml

```diff
@@ -48,7 +48,7 @@ jobs:
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@v4
         with:
-          title: 'Release ${{ steps.cz.outputs.version }}'
+          title: 'chore: release ${{ steps.cz.outputs.version }}'
```

Comment:
Make sure the release PR title matches commitizen so we don't accidentally create commit messages that don't match the spec